.buildkite/scripts/steps/functional/performance_playwright.sh

```diff
@@ -33,7 +33,8 @@ unset ELASTIC_APM_GLOBAL_LABELS
 export TEST_ES_URL=http://elastic:changeme@localhost:9200
 export TEST_ES_DISABLE_STARTUP=true
 
-sleep 120
+# Pings the es server every seconds 2 mins until it is status is green
+curl --retry 120 --retry-delay 1 --retry-all-errors -I -XGET "${TEST_ES_URL}/_cluster/health?wait_for_status=green"
```

Comment:
The approach sounds good to me. Waiting for cluster health "green" might be too restrictive though. We should instead align the approach with the [Elasticsearch tests](https://github.com/elastic/elasticsearch/blob/master/build-tools/src/main/java/org/elasticsearch/gradle/testclusters/WaitForHttpResource.java#L59) by specifying `${TEST_ES_URL}/_cluster/health?wait_for_nodes=>=1&wait_for_status=yellow"` as URL.

Reply:
Definitely makes sense however not sure how long does it take yellow to green transitions on CI. Locally I haven't even seen status to be yellow it immediately becomes green so it's pretty fast to become ready. I'm a bit hesitant though if it hangs on yellow more than expected kibana instance will start but will crash because it doesn't get a response from ES and pipeline will fail.

Reply:
As discussed offline, a cluster in `yellow` status could be perfectly fine (e.g. a single node cluster containing an index with `replicas` set to `1`) and we won't have full control over that because Elasticsearch can add indices on its own upon startup. Therefore, it makes sense to align the health status check as I've described above.